src/bin/pg_autoctl/cli_do_monitor.c

```diff
@@ -223,17 +223,16 @@ cli_do_monitor_get_other_nodes(int argc, char **argv)
 	{
 		char json[BUFSIZE];
```

Comment:
this has become useless after the change

Reply:
Thanks, good catch. Fixed.